CLI/commands/sto_manager.js

```diff
@@ -583,28 +645,52 @@ async function usdTieredSTO_status(currentSTO) {
     let balance = await getBalance(displayWallet, gbl.constants.FUND_RAISE_TYPES[type]);
     let walletBalance = web3.utils.fromWei(balance);
     let walletBalanceUSD = web3.utils.fromWei(await currentSTO.methods.convertToUSD(gbl.constants.FUND_RAISE_TYPES[type], balance).call());
```

Comment:
You should check the balance for each stable coin. For example if POLY, ETH, DAI and USDT are accepted, you should show
Balance ETH: xx ETH
Balance POLY: xx POLY
Balance DAI: xx DAI
Balance USDT: xx USDT

Reply:
Waiting SC update